molecule/default/molecule.yml

```diff
@@ -3,33 +3,121 @@ dependency:
   name: galaxy
 driver:
   name: docker
-lint:
-  name: yamllint
+lint: |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
 platforms:
-  - name: amazonlinux2
-    image: amazonlinux:2
-  - name: debian9
-    image: debian:stretch-slim
-  - name: debian10
-    image: debian:buster-slim
-  - name: fedora29
-    image: fedora:29
-  # Ansible currently insists on installing python2-dnf, which does
-  # not exist in Fedora 30.  Until that is resolved, we can't use
-  # Fedora 30.
-  # - name: fedora30
-  #   image: fedora:30
-  - name: fedora31
-    image: fedora:31
-  - name: kali
-    image: kalilinux/kali-rolling
+  - name: amazonlinux2_systemd
```

Comment:
This role doesn't need systemd to actually be running.  It is only editing the file `/etc/systemd/journald.conf`, so you could just as well use the non-systemd Docker images here but install the `systemctl` package as part of the `prepare.yml` runbook.  This is how the role used to work.

I personally prefer to avoid the systemd-enabled images when possible, since they run with extra permissions and restart my X session on my local machine.  There may be a good argument for using them here, so I'm open to discussion.